frontend/src/components/activity/content/StoryboardRow.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <component
+    :is="variant !== 'dense' ? 'tr' : 'div'"
+    :role="variant === 'dense' ? 'row' : null"
+    class="e-storyboard-row"
+    :class="{
+      'e-storyboard-row--default': variant === 'default',
+      'e-storyboard-row--dense': variant === 'dense',
+    }"
+  >
+    <component
+      :is="variant !== 'dense' ? 'td' : 'div'"
+      :role="variant === 'dense' ? 'cell' : null"
+      class="e-storyboard-row__handle"
+    >
+      <v-btn
+        icon
+        small
+        class="drag-and-drop-handle"
+        :disabled="isLastSection"
+        :aria-label="$tc('components.activity.content.storyboardRow.move')"
+        @keydown.down="$emit('moveDown', itemKey)"
```

Comment:
Ah I see, here are the keyup/keydown. Definitely a "power-user" thing and didn't work well for me even after knowing the feature. I thought we wanted to keep the arrows especially for mobile, where this hack doesn't work. But also for this, I'm ok to merge and we can improve/discuss later.

Reply:
I agree. I think this power feature would need some work anyway, because if you move an entry down, the focus should stay on the handle. We could also show a hint when the handle ist focused that you can move the entry with the arrow keys.

We tried tried drag&drop on different mobile platforms and were surprised that it worked so well. So we decided to use the arrows only on focus. But in my experiments I made an overlay over the whole row that was only visible when focused and this felt strange and cumbersome, as it introduced two tabstops per row that are initially hidden. I think we can find out in the user feedback.